### PR TITLE
[YARP] [MAS] Adding support for multiple analog sensor interface methods

### PR DESCRIPTION
**Pull request by prashanthr05**
Fixes [#13](https://github.com/robotology-playground/xsensmt-yarp-driver/issues/13) Multiple analog sensor interface methods for xsensmt driver

### CODE REVIEW

**Comment by prashanthr05** [comment]
@gabrielenava @traversaro

**Comment by prashanthr05** [comment]
The action over making parameters `sensor_name` and `frame_name` default will be taken soon in a future commit.

File: xsensmt/XsensMT.cpp [hunk 1]
```diff
@@ -322,3 +325,195 @@ bool XsensMT::setOptionFlags(const uint32_t setFlags, const uint32_t clearFlags)
...
+}
+
+bool XsensMT::getThreeAxisMagnetometerMeasure(size_t sens_index, yarp::sig::Vector& out, double& timestamp) const
+{
+    if (sens_index != 0 || m_isSensorMeasurementAvailable == false)
+    {
+        return false;
+    }
+    
+    out.clear();
```
Comment (traversaro) [inline xsensmt/XsensMT.cpp]: No need to do clear: anything that is in the vector after the `resize(3)` will be overwritten.

Comment (prashanthr05) [reply xsensmt/XsensMT.cpp]: Fixed. Thanks!

**Comment by prashanthr05** [comment]
Fixed the parameter handling according to comment [#1910](https://github.com/robotology/yarp/pull/1910#discussion_r230317906) from YARP.

**Comment by prashanthr05** [comment]
@traversaro whenever you have time, could you check this PR ?

**Comment by traversaro** [comment]
Thanks @prashanthr05 , sorry for the delay. 
@gabrielenava I guess it is ok for you if we merge this.

**Comment by prashanthr05** [comment]
@traversaro Thanks a lot!

**Pull request merged**